.github/workflows/local-pr.yaml

```diff
@@ -0,0 +1,171 @@
+name: "Local PR"
+run-name: "Local PR requested for #${{ github.event.issue.number }} by @${{ github.actor }}"
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  pr_commented:
+    name: Identify comment source
+    if: ${{ github.event.issue.pull_request }} || ${{ github.event.issue.state == 'open' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment came from Pull Request
+        run: echo "Pull request check succeeded."
+
+  check_user_permission:
+    runs-on: ubuntu-latest
+    name: Check permissions
+    needs: [pr_commented]
+    steps:
+      # Check for write permission
+      - name: Check write permission
+        id: check
+        uses: actions-cool/check-user-permission@v2
```

Comment:
We probably want to review this for a) what it does with the token and b) if it really checks for write permissions properly on the repo for the supplied (or currently default) user.

Reply:
It is not required to supply a PAT for this action.

Reply:
I've removed the PAT from the permission check and added a test to the testing repository for correct permission checking.